Makefile

```diff
@@ -175,6 +175,15 @@ setup-contract-tests-data:
 	./build/gaiad init --home /tmp/contract_tests/.gaiad --chain-id lcd contract-tests ; \
 	tar -xzf lcd_test/testdata/state.tar.gz -C /tmp/contract_tests/
 
+	# speed up tests by reducing timeouts
```

Comment:
How do these additions fix the original problem? Also, if this is find-and-replace, it seems pretty hacky and fragile.

Reply:
I agree with @alexanderbez here - wouldn't it be better to just provide a `config.toml`?

Reply:
We had a `config.toml` inside `state.tar.gz`, I removed it in order to avoid this problem to repeat.
The pipeline was not working because `db_backend` changed from `leveldb` to `goleveldb` and when the script was replacing the generated `config.toml` with the one inside `state.tar.gz` this change got overridden